### PR TITLE
Ruby build matrix updates

### LINKED ISSUE #164: Add jruby 9.x to travis builds.
Apparently we're not doing this?

### PR DESCRIPTION
**Pull request by JonRowe**
Fixes #164 and #160

### CODE REVIEW

File: travis/.travis.yml [hunk 1]
```diff
@@ -32,11 +33,14 @@ matrix:
       env: JRUBY_OPTS='--server -Xcompile.invokedynamic=false'
     - rvm: jruby
       env: JRUBY_OPTS='--server -Xcompile.invokedynamic=false'
+    - rvm: jruby-9.0.3.0
```
Comment (myronmarston) [inline travis/.travis.yml]: Why 9.0.3.0 instead of 9.1.2.0?

Comment (JonRowe) [inline travis/.travis.yml]: To be honest, because that was the latest version I found, oops!

**Comment by JonRowe** [comment]
Updated across the PRs, rspec/rspec-support#291 is the most ready to be merged (and prerequisite of the others), mocks and core builds need some more tweaking which I'll try to do today

**Comment by JonRowe** [comment]
Oh and rspec/rspec-expectations#930 is also good to go having had the least problems!

**Comment by JonRowe** [comment]
Only rspec/rspec-core#2293 outstanding, it's green with caveats

**Comment by JonRowe** [comment]
Merging as all PRs are merged.

**Pull request merged**